examples/pacman.md

```diff
@@ -34,5 +34,54 @@ Searching for name...
 ```
 *(users can "stack" short subcommands with short flags or with other short flag subcommands)*
 
+In the help, this looks like:
+```console
+$ pacman -h
+pacman 5.2.1
+Pacman Development Team
+package manager utility
+
+USAGE:
+    pacman[EXE] <SUBCOMMAND>
+
+OPTIONS:
+    -h, --help       Print help information
+    -V, --version    Print version information
+
+SUBCOMMANDS:
+    -Q--queryquery    Query the package database.
```

Comment:
@epage This doesn't look correct to me.

Reply:
Agreed.  I just didn't want to block getting this documented for users on fixing the behavior.  Was going to do this in a follow up though.